File: pozostale/cpp/zgadywanka.cpp
#include <iostream>
#include <stdlib.h>
using namespace std;


int main() {
	int liczba = rand() % 1000000000000 + 1;
    int strzal;
    cout << "Zgadnij liczbę od 1 do 1000000000000: ";
    while (strzal != liczba){
        cin >> strzal;
        if (strzal < liczba){
            cout << "Za mało!\nSpróbuj jeszcze raz: ";
        }
        if (strzal > liczba){
            cout << "Za dużo!\nSpróbuj jeszcze raz: ";
        }
    }
    cout << "Zgadłeś!\n";
}

File: pozostale/python/zgadywanka.py
from random import randint


def zgaduj():
    n = randint(0, 10000)
    k = -1

    while k != n:
        k = int(input("Zgadnij liczbę od 0 do 100000: "))
        if k < n:
            print("Za mało!")
        elif k > n:
            print("Za dużo!")

    print("Zgadłeś! To była liczba", n)


if __name__ == "__main__":
    zgaduj()
